### PR TITLE
Additioanl keys

### PR DESCRIPTION
**Pull request by xkbeyer**
More support for edge, since there the keys have different names.
For debugging I add key '+' to set the speed to max.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -91,12 +91,14 @@ window.addEventListener('touchstart', raise_longcat);
 // Some cheats for testing
 window.addEventListener('keyup', (ev) => {
   ev.preventDefault();
-  if (ev.key == 'ArrowUp') {
+  if (ev.key == 'ArrowUp' || ev.key == 'Up') {
```
Comment (ThibaultJanBeyer) [inline lib/main.js:94]: 👍

File: lib/main.js [hunk 2]
```diff
@@ -91,12 +91,14 @@ window.addEventListener('touchstart', raise_longcat);
...
     the_physics.g = 0;
     the_physics.vy += 0.01;
-  } else if (ev.key == 'ArrowDown') {
+  } else if (ev.key == 'ArrowDown' || ev.key == 'Down') {
     the_physics.g = 0;
     the_physics.vy -= 0.01;
+  } else if (ev.key == '+' || ev.key == 'Add') {
+    the_physics.vy = 1;
```
Comment (ThibaultJanBeyer) [inline lib/main.js:101]: let’s try this 🥇 💯

Comment (lucasb-eyer) [inline lib/main.js:101]: :rocket:

**Review by ThibaultJanBeyer — APPROVED** [review]
tumblr_oatqjozea01rn7bzro1_400

**Pull request merged**